ocs_sample_library_preview/SDS/SdsStreamView.py

```diff
@@ -7,162 +8,176 @@ class SdsStreamView(object):
     Sds StreamView definitions
     """
 
+    def __init__(self, id: str = None, name: str = None, description: str = None,
+                 source_type_id: str = None, target_type_id: str = None,
+                 properties: list[SdsStreamViewProperty] = None):
+        """
+        :param id: required
+        :param name: not required
```

Comment:
same comment as earlier about the required/optional parameter ordering

Reply:
This has been fixed.